server/route.js

```diff
@@ -77,6 +79,7 @@ Route = function(router, path, options) {
       var originalWrite = res.write;
       res.write = function(data) {
         data = data.replace('</head>', page.head + '\n</head>');
+        data = moveScripts(data);
```

Comment:
I think we need to do this, if only the user asked to do it.
Just check `FlowRouter.deferScriptLoading` option is enabled or not.

Add a new function called `FlowRouter.setDeferScriptLoading` which adds following option.